src/speechly/client.ts

```diff
@@ -189,6 +191,10 @@ export class Client {
    * @param cb - the callback which is invoked when the context start was acknowledged by the API.
    */
   async startContext(): Promise<string> {
+    if (this.resolveStopContext != null) {
+      this.resolveStopContext()
```

Comment:
This will not work reliably, because we only signal the stop code to proceed but we don't `await` for stop to actually happen.

Reply:
await added.